rma/models/stock_move.py

```diff
@@ -10,30 +11,24 @@ class StockMove(models.Model):
 
     # RMAs that were created from the delivery move
     rma_ids = fields.One2many(
-        comodel_name="rma",
-        inverse_name="move_id",
-        string="RMAs",
+        "rma",
```

Comment:
Don't change this. Useless diff and doing it less explicit removing keyword arguments.